default_config.json

```diff
@@ -30,6 +30,113 @@
         "PL",
         "RDS"
     ],
+    "filter.object_types": [
+        "Gx",
+        "OC",
+        "Gb",
+        "Nb",
+        "DN",
+        "PN",
+        "C+N",
+        "Ast",
+        "Kt",
+        "***",
+        "D*",
+        "*",
+        "?",
+        "Pla",
+        "CM"
+    ],
+    "filter.constellations": [
```

Comment:
I have a PR which has a more generic way of filtering, ie you can filter on any attribute of an object - when I get to it again I'll have to refactor,  but just a heads up that we don't have to hard code everything

Reply:
I'm curious what your filter refactor will look like here.  Does it operate on the principle of an empty list = all?

It used to be that an empty list (or None) of object types meant 'all', but I'm intentially switching the filtering system away from this as it required multiple exceptions and made the typing a bit strange.  Having the explicit list in the default_config is a nice way to know for sure what resetting the filters does.